content/projects/gsoc/2024/project-ideas.html.haml

```diff
@@ -0,0 +1,161 @@
+---
+layout: simplepage
+title: "GSoC 2024 Project Ideas"
+year: 2024
+tags:
+- gsoc
+- gsoc2024
+project: gsoc
+opengraph:
+  image: /images/gsoc/opengraph.png
+---
+
+//The individual page's layout is located at /content/_layouts/gsocprojectidea.html.haml
+
+:css
+  .pi_table {
+    width: 100%;
+  }
+  .pi_table_header {
+    text-align: center;
+  }
+  .pi_table_category {
+    width: 100px;
+  }
+  .pi_table_skills {
+    width: 175px;
+  }
+
+%img{:title => "Jenkins GSoC", :src => expand_link("/images/gsoc/jenkins-gsoc-logo_small.png"), :style => "float:right", :width => "128"}
+
+
+%p
+  = succeed '.' do
+    This page aggregates project ideas for Google Summer of Code
+    = page.year
+  = succeed '.' do
+    See more information about this project and applications
+    %a{:href => expand_link("/projects/gsoc/")}
+      on the Jenkins Google Summer of Code page
+%p
+  Below you can find project ideas which have been proposed for this year.
+  New ideas may be proposed by interested mentors or GSoC contributors (e.g. new features in the core, "write a plugin for MY_TOOL_OR_SERVICE", etc.).
+  Project ideas without potential mentors will be considered though applicants may need to work with the community and GSoC org admins to find mentors.
+  To add a new project idea, see:
+  = succeed '.' do
+    %a{:href => expand_link("/projects/gsoc/proposing-project-ideas/")}
+      proposing project ideas
+
+%h3
+  Accepted ideas
+
+%p
+  Below you can see the list of project ideas that fully match the Jenkins' project idea standard.
+  The scope of these ideas is well known and we don't normally expect deep changes.
+  All ideas have quick start guidelines and newbie-friendly issues referenced.
+  We welcome contributors to join the mentor teams, and we invite GSoC contributors to submit project proposal applications in relation to these ideas.
+
+%table.pi_table.grid-all
+  %thead
+    %tr.pi_table_header
+      %th Project
+      %th Category
+      %th Skills to study/improve
+  %tbody
+    // TODO: parameterize year
+    - ideas = site.pages.select { |p| p.source_path =~ /\/content\/projects\/gsoc\/2024\/project-ideas\/[^\/]+.adoc/ }
+    - ideas.each do |item|
+      - if item.status == "published"
+        %tr
+          %td
+            %a{:href => item.url}
+              %strong
+                = item.title
+            %br
+            = succeed '.' do
+              = item.goal
+            %br
+            %b
+              Potential Mentor(s):
+            - item.mentors.each do |mentor|
+              = display_user_optional(mentor)
+          %td.pi_table_category
+            - if item.category
+              = item.category
+            - else
+              Undefined
+          %td.pi_table_skills
+            = item.skills.join(", ")
+
+%h3
+  Draft project ideas
+
+%p
+  Below you can see draft project ideas, which are currently under review.
```

Comment:
```suggestion
  In the following list, you can refer to draft project ideas, which are currently under review.
```
Adjustment for syntax as `Below` is [not recommended in this context](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/b/below). The suggestion to use `In the following list` replaces below but still provides clarity & direction for where the reader should look.